index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+<body>
+  <header class="header">
+      <a>Arrow Dan</a>
+      <nav class="navbar container">
+        <img class="hamburger" src="image/Union.png" alt="navigation icon">
+      </nav>
+  </header>
+    <div class="container">
+        <h2 class="logo-message">I'm  Dan</h2>
+        <h3 class="dev-intro">I'am a Software Developer</h3>
+        <p class="background">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+    </div>
+    <div class="social-media  container">
+        <img src="image/github.png" alt="github icon">
+        <img src="image/linkedin.png" alt="linkedin icon">
+        <img src="image/finger.png" alt="peace finger sign">
+        <img src="image/twitter.png" alt="twitter icon">
+        <img src="image/m.png" alt="capital M">
+    </div>
+    <img src="image/illustration-header-mobile.png" alt="a lovely corner image" class="corner-image">
```

Comment:
- [ ] Kindly use `background-image: url("img url");` role in your CSS instead of `img` tag, because decorations are always background images, not `img` tags.

Reply:
- [ ] Please kindly implement this required change